.circleci/config.yml

```diff
@@ -346,7 +361,14 @@ workflows:
             branches:
               only:
                 - release
+
+      - smoke-test-on-live-env:
+          filters:
+            branches:
+              only:
                 - staging
+          requires:
+            - hokusai/deploy-staging
```

Comment:
I don't think we want this, since the `hokusai/deploy-staging` step runs [at the end of the `master` workflow](https://app.circleci.com/pipelines/github/artsy/force/21700/workflows/cd9710c6-690a-4da0-b589-da6bf4578646) and updates the `staging` branch itself.

Reply:
make sense, just removed this dependency https://github.com/artsy/force/pull/8913/commits/3c4a514292ac66cb86d7e51b2ddfdefecea6ceb8